plugins/kotlin-dataframe/testData/diagnostics/selectDuringTyping.fir.txt

```diff
@@ -28,9 +28,8 @@ FILE: selectDuringTyping.kt
             }
 
             local abstract class ExplodeSchema_94 : R|<local>/ExplodeSchema_94I| {
-                public abstract var scope0: R|<local>/Scope0|
+                @R|org/jetbrains/kotlinx/dataframe/annotations/ScopeProperty|() public abstract val scope0: R|<local>/Scope0|
                     public get(): R|<local>/Scope0|
```

Comment:
should the fir files in diagnostics also be included, or are they fine?

Reply:
For diagnostics tests, FIR dumps are needed to verify that warnings/errors were actually reported, it seems